src/statd/python/yanger/yanger.py

```diff
@@ -664,6 +664,15 @@ def add_ip_link(ifname, iface_in, iface_out):
     if 'link' in iface_in and not iface_is_dsa(iface_in):
         insert(iface_out, "infix-interfaces:vlan", "lower-layer-if", iface_in['link'])
 
+    if 'flags' in iface_in:
+        admin_xlate = { 
+            "UP": "up", 
+            "DOWN": "down" 
+            }
+
+        admin_status = admin_xlate.get("UP" if "UP" in iface_in['flags'] else "DOWN", "unknown")
```

Comment:
Please replace `"unknown"` with `"testing"`, since unknown is not listed in the enum for admin-status in the ietf-interfaces YANG model.

Reply:
Oh, yes. Thnx. Now I remember seeing it in the yang model